backend/molgenis-emx2-graphql/src/test/java/org/molgenis/emx2/graphql/TestGraphqlAdminFields.java

```diff
@@ -57,6 +60,83 @@ public void testUsers() {
         });
   }
 
+  @Test
+  void testUpdateUser() {
+    database.tx(
+        testDatabase -> {
+          testDatabase.becomeAdmin();
+          GraphQL graphql = new GraphqlApiFactory().createGraphqlForDatabase(testDatabase, null);
+
+          try {
+            // setup
+            testDatabase.dropCreateSchema(schemaName);
+            testDatabase.dropCreateSchema(ANOTHER_SCHEMA_NAME);
+            testDatabase.addUser(TEST_PERSOON);
+            testDatabase.setEnabledUser(TEST_PERSOON, true);
+            testDatabase.getSchema(schemaName).addMember(TEST_PERSOON, "Owner");
+            testDatabase.getSchema(ANOTHER_SCHEMA_NAME).addMember(TEST_PERSOON, "Viewer");
+
+            // test
+            String query =
+                "mutation updateUser($updateUser:InputUpdateUser) {updateUser(updateUser:$updateUser){status, message}}";
+            Map<String, Object> variables = createUpdateUserVar();
+            ExecutionInput build =
+                ExecutionInput.newExecutionInput().query(query).variables(variables).build();
+            String queryResult = convertExecutionResultToJson(graphql.execute(build));
+            JsonNode node = new ObjectMapper().readTree(queryResult);
+            if (node.get("errors") != null) {
+              throw new MolgenisException(node.get("errors").get(0).get("message").asText());
+            }
+
+            // assert results
+            User user = testDatabase.getUser(TEST_PERSOON);
+            assertEquals("testPersoon", user.getUsername());
+            assertFalse(user.getEnabled());
+
+            List<Member> members = testDatabase.getSchema(schemaName).getMembers();
+            assertTrue(members.isEmpty());
+
+            Member anotherSchemaMember =
+                testDatabase.getSchema(ANOTHER_SCHEMA_NAME).getMembers().stream().findFirst().get();
+            assertEquals("Owner", anotherSchemaMember.getRole());
+            assertEquals(TEST_PERSOON, anotherSchemaMember.getUser());
+
+            // clean up
+            testDatabase.removeUser(TEST_PERSOON);
```

Comment:
I would keep the schema, generally we don't cleanup schemas because it helps debugging.

Reply:
Changed it, but shouldn't unit test run in a vacuum? I kind of dislike that running the tests clogs up my database with schemas.